frappe/integrations/doctype/ldap_settings/test_ldap_settings.py

```diff
@@ -1,10 +1,690 @@
 # -*- coding: utf-8 -*-
 # Copyright (c) 2019, Frappe Technologies and Contributors
 # See license.txt
+<<<<<<< HEAD
 from __future__ import unicode_literals
 
 # import frappe
+=======
+import frappe
+>>>>>>> 172d1b31e8 (test: LDAP unit-tests)
```

Comment:
fix:

cherry-pick b23b53d889e1b8c70850797bf00ae3968767f61b conflict originating from 172d1b31e8b1abe7c2792d97456705dc4021c261. 
- unit tests should now pass
- fixes sider `syntax` errors

```suggestion

import frappe
```